frontend/test/lit/test_function_attributes.py

```diff
@@ -28,7 +28,7 @@ def qnode(x):
 
 @qjit(target="mlir")
 # The entry point has no internal linkage.
-# CHECK-DAG: func.func public @jit.workload(%arg0: tensor<f64>) -> tensor<4xcomplex<f64>> {
+# CHECK-DAG: func.func public @jit.workload(%arg0: tensor<f64>) -> tensor<4xcomplex<f64>> attributes {llvm.emit_c_interface} {
```

Comment:
The patch introduces this change. This is because the target="mlir" would not have added the attributes since `append_modules` was executed in a different time.

Reply:
Got it 👍